oblivious-lru-cache.d.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Mnemonist ObliviousLRUCache Typings
+ * ===========================
+ */
+ import {IArrayLikeConstructor} from './utils/types';
+
+ export default class ObliviousLRUCache<K, V> implements Iterable<[K, V]> {
```

Comment:
I may be wrong but I think you can import the `LRUCache` class in this definition file and make this class definition `extends` it, no?